Automattic/jetpack
# Payments Block: improve redirection auto open modals

Fixes #23647

#### Changes proposed in this Pull Request:

* We remove a parameter from the URL before we open the payment form so that future reloads of the page (if any) will not trigger the form to open again.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create or edit a post with a Payment Block.
* Open the email and/or notification and click on the payment link.
* You should be redirected to the post and the form should open.
* The URL should not contain the param that controls if the form should open.

#### Test evidence
https://user-images.githubusercontent.com/1989914/160827627-1621d28e-8fdc-4204-97be-757a1c563a79.mp4
